panels/configspec.ini

```diff
@@ -61,3 +61,4 @@ pending = string(default="Pending")
 accepted = string(default="Accepted")
 declined = string(default="Declined")
 waitlisted = string(default="Waitlisted")
+cancelled = string(default="Cancelled")
```

Comment:
This status indicates, "This panel was accepted, but then the panelist backed out, so it's not happening."  We don't want to use "Declined" for that, since we'd want to send a different email (which will probably not be automated since this is a very unique snowflake kind of situation).